src/components/ProgramForm/ThreeColProgram/List/Recommender/ExerciseRecommender.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormattedMessage } from 'umi';
+import { Button } from 'antd';
+import Paragraph from 'antd/lib/typography/Paragraph';
+
+interface ExerciseRecommenderProps {
+  recommendedData: API.RecommenderExercise;
+  createTopic: () => void;
+}
+
+export const ExerciseRecommender: React.FC<ExerciseRecommenderProps> = ({
+  recommendedData,
+  createTopic,
+}) => {
+  const createText = (topic: API.RecommenderTopicType) =>
+    topic?.includes('.') && topic?.split('.').join(' ');
```

Comment:
Move it outside of component scope, it's pure function. I suggest adding return type here.